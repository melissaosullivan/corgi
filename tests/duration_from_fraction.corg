duration d = $1/4$;
frac f = $3/4$;
duration fd = duration(f);