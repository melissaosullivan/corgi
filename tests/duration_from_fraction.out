
int main()
{
duration d = $1/4$;
frac f = $3/4$;
}

